api/v1beta1/zz_generated.deepcopy.go

```diff
@@ -123,7 +122,8 @@ func (in *AuthConfigSpec) DeepCopyInto(out *AuthConfigSpec) {
 			if val == nil {
 				(*out)[key] = nil
 			} else {
-				in, out := &val, &outVal
+				inVal := (*in)[key]
```

Comment:
these changes should not be here. Checkout go version. It should be 1.21.

Then, delete `bin` directory and run `make generate`. There should not be any change in this generated files.

Reply:
I still see the changes